assets/components/atoms/breadcrumbs/breadcrumbs.scss

```diff
@@ -0,0 +1,57 @@
+@charset 'utf-8';
+
+.breadcrumb {
+  .breadcrumb-item {
+    display: inline-block;
+    position: relative;
+    font-size: 0.92rem;
+    line-height: 1.5rem;
+
+    &:before {
+      content: '';
+      display: none;
+    }
+
+    &:after {
+      content: '>';
```

Comment:
`>` is not very elegant. The proper character is `›`, but use UTF-8 code in `content` as much as possible, it's safer. In your case, it's something like `content: "\203A";` You can find any character code in [charbase.com](http://www.charbase.com/block) 😉 

![breadcrumb_et_styleguide](https://user-images.githubusercontent.com/2025924/36414428-88390454-1622-11e8-88ac-8a65bacc0e76.png)

Reply:
Thanks for the detailed review, I just discovered a new character 😅 it was meant to be a placeholder until we got the svg. Fixed!